agogosml_cli/cli/templates/{{cookiecutter.PROJECT_NAME_SLUG}}/e2e-ci-pipeline.yml

```diff
@@ -0,0 +1,50 @@
+resources:
+- repo: self
+
+variables:
+  container_registry: '{{cookiecutter.AZURE_CONTAINER_REGISTRY}}'
+  azure_subscription: '{{cookiecutter.SUBSCRIPTION_ID}}'
```

Comment:
I'll create another PR that deals with
1. moving these to variable groups
2. updates docs across the different areas that we're no using variable groups.